editoast/src/models/fixtures.rs

```diff
@@ -29,6 +30,7 @@ use crate::models::Study;
 use crate::models::Tags;
 use crate::views::rolling_stock::form::RollingStockForm;
 use crate::views::train_schedule::TrainScheduleForm;
+use crate::views::work_schedules::WorkScheduleItemForm;
```

Comment:
In prevision of the editoast split in crates, we don't want to use `crate::views` in `models` (it's the opposite we want). The other `use crate::views` shouldn't be there and will need to be removed at some point. Even conceptually, the `XyzForm`-like structs represent views exchange format and don't have anything to do in the `models` module.

Just use a `Changeset<WorkSchedule>` instead.

Reply:
alright, done